src/topics/TopicListScreen.js

```diff
@@ -2,19 +2,26 @@
 
 import React, { PureComponent } from 'react';
 
-import type { Dispatch, GlobalState, Stream, TopicExtended } from '../types';
-import { connectFlowFixMe } from '../react-redux';
+import type { NavigationScreenProp } from 'react-navigation';
+import type { Dispatch, Stream, TopicExtended } from '../types';
+import { connect } from '../react-redux';
 import { Screen } from '../common';
 import { topicNarrow } from '../utils/narrow';
 import { getTopicsForStream } from '../selectors';
 import { getStreamForId } from '../subscriptions/subscriptionSelectors';
 import TopicList from './TopicList';
 import { fetchTopics, doNarrow } from '../actions';
 
+type SelectorProps = $ReadOnly<{|
+  stream: Stream,
+  topics: ?(TopicExtended[]),
```

Comment:
nit: commit message says `TopicExtended` where it means `TopicExtended[]`. Also best to parenthesize `?(TopicExtended[])`, same as in the code. (I'm not sure if `?TopicExtended[]` means the same thing, or means `(?TopicExtended)[]`.)